Description: Patch adds missing headers which helps building BLT on
    64-bit architectures.
Author: Matthias Klose
Last-Modified: Sun, 06 Jul 2014 07:28:14 +0400

--- a/generic/bltNsUtil.c
+++ b/generic/bltNsUtil.c
@@ -27,6 +27,8 @@
 
 #include "bltInt.h"
 #include "bltList.h"
+#include "bltNsUtil.h"
+#include "tclInt.h"
 
 /* Namespace related routines */
 
--- a/configure.in
+++ b/configure.in
@@ -825,6 +825,7 @@
 	"${TCL_INC_DIR}" != "${TK_INC_DIR}" ; then
   INC_SPECS="${INC_SPECS} -I${TCL_INC_DIR}"
 fi
+INC_SPECS="${INC_SPECS} -I${TCL_SRC_DIR}/generic -I${TCL_SRC_DIR}/unix"
 
 
 # On Windows, override the default include directory with our own.
